src/core/Akka/Serialization/Serializer.cs

```diff
@@ -121,7 +121,17 @@ public static int GetSerializerIdentifierFromConfig(Type type, ExtendedActorSyst
         {
```

Comment:
At some point Typesafe guys added ability to specify identifier for each serializer directly through config i.e. for case, when you have conflicting serializers.